src/dev/flang/ast/AbstractType.java

```diff
@@ -1442,6 +1440,28 @@ public AbstractType checkConstraints(SourcePosition pos)
   }
 
 
+  /**
+   * Position in source where this type is defined:
+   *
+   * 1) internally defined types like universe, t_Error,
+   *    fuzion.sys.internal_array, unit etc.
+   *    result in SourcePosition.notAvailable
+   * 2) type is generic argument: some_feat(A type) is
+   *                                     ---^
+   * 3) type belongs to feature : some_feat is
+   *                           ---^
+   */
+  public SourcePosition pos()
```

Comment:
maybe we should name this `posOfDef()` or similar to not have it confused with other `pos()`